vividus-plugin-applitools/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -37,7 +37,7 @@
         factory-bean="httpClientFactory"
         factory-method="buildHttpClient">
         <constructor-arg>
-            <bean parent="restApiHttpClientConfig">
+            <bean parent="httpClientConfig">
```

Comment:
In this case I believe the configuration of interceptors and handlers becomes redundant